content/v2.0/write-data/no-code/_index.md

```diff
@@ -0,0 +1,19 @@
+---
+title: No-code solutions
+seotitle: Write data to InfluxDB without coding
+list_title: Write data to InfluxDB without coding
+weight: 101
+description: >
+  Write data to InfluxDB without writing code.
+aliases:
+  - /v2.0/collect-data/advanced-telegraf
+  - /v2.0/collect-data/use-telegraf
+menu:
+  v2_0:
+    name: No-code solutions
+    parent: Write data
+---
+
+The following options let you write data to InfluxDB without writing any code.
```

Comment:
We may want to indicate that _configuration_ may be required, but it should be fairly minimal.